src/istgt_ser_table.c

```diff
@@ -93,24 +95,44 @@
 
 istgt_serialize_action
 istgt_serialize_table[ISTGT_SERIDX_COUNT + 1][ISTGT_SERIDX_COUNT + 1] = {
-/**>IDX_ :: 2nd:TUR RD  WRT UNMP MDSN MDSL RQSN INQ  RDCP RES  REL LSNS FMT STR PRIN PROT MAININ INVLD*/
-/*TUR     */{   pS, pS, pS,  pS,  pS,  pS,  pS,  pS,  pS,  bK,  bK, pS,  pS, pS, bK,  bK,  pS,    pS},
-/*READ    */{   pS, pS, xT,  bK,  bK,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*WRITE   */{   pS, xT, xT,  bK,  bK,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*UNMAP   */{   pS, bK, bK,  pS,  bK,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, pS,  bK,  bK,    pS},
-/*MD_SNS  */{   pS, bK, bK,  bK,  pS,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*MD_SEL  */{   pS, bK, bK,  bK,  bK,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*RQ_SNS  */{   pS, pS, pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS, pS,  pS, pS, pS,  pS,  pS,    pS},
-/*INQ     */{   pS, pS, pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS, pS,  pS, pS, pS,  pS,  pS,    pS},
-/*RD_CAP  */{   pS, pS, pS,  pS,  pS,  pS,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*RESV    */{   bK, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*REL     */{   bK, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*LOG_SNS */{   pS, pS, pS,  pS,  pS,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*FORMAT  */{   pS, bK, bK,  bK,  bK,  bK,  pS,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*START   */{   pS, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*PRES_IN */{   bK, bK, bK,  pS,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*PRES_OUT*/{   bK, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*MAIN_IN */{   pS, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  pS,    pS},
-/*INVLD   */{   pS, pS, pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS, pS,  pS, pS, pS,  pS,  pS,    pS}
+/*
+ * >IDX_ :: 2nd:TUR RD  WRT UNMP MDSN MDSL RQSN INQ  RDCP RES  REL LSNS
```

Comment:
For easy reading of the table, can you please change these lines as
```
/*
 * >IDX_ :: 2nd:
 * TU RD WRT UNMP MSN MDS RQS INQ RDC RES REL LSN FMT STR PR PROT MAI INV
 * R                  L   N       P           S           IN      NIN LD
 */
```

Reply:
Updated the comment as suggested.